app/stac_api/collection_temporal_extent.py

```diff
@@ -173,6 +199,9 @@ def update_end_temporal_extent_on_item_update(
             the id of the item being treated (pk)
         qs: QuerySet | None
             queryset with all items of the collection excluding the one being updated. (optional)
+        qs_other_items_with_properties_datetimes: QuerySet | None
+            Already evaluated queryset with all items (but the one currently updated) that have
```

Comment:
Is not necessary already evaluated if used with exists() see comment above

Reply:
I am not sure if I fully understand your point here.

My reasoning for passing the `qs_other_items_with_properties_datetime` as an optional argument to `update_end_temporal_extent_on_item_update()` is, that probably this qs has already been evaluated earlier on in `update_start_temporal_extent_on_item_update`.

So `update_start_temporal_extent_on_item_update` returns either this very qs or None. If it has returned the qs, then this qs is already evaluated and doesn't need to be evaluated again in `update_end_temporal_extent_on_item_update`.
So we can save on DB hit here, for items, where both, starts and ends have been udated.

Similar implementation is done for updating on item_delete instead of item_update further below.